### PR TITLE
Modified the utils and the demo.ts file to showcase the functionality of updating the stream

### PR DESCRIPTION
**Pull request by belyrs**

### CODE REVIEW

**Review by amarts — APPROVED** [review]
Please note. Any changes inside of SDK should be called out, because it needs a release to get reflected for those who want to run this.

File: packages/modules/src/document/Document.ts [hunk 1]
```diff
@@ -371,7 +371,7 @@ export async function fromContent({
   return document
 }
 
-export async function updateStream(
+export async function updateFromContent(
```
Comment (smohan-dw) [inline packages/modules/src/document/Document.ts]: Any reason we are not following the same function style as fromContent here? Also, the update will be either at the content level or the options - like the expiry date change. If the content is not changed, we don't need to process the content. Also, if the content is changed, we should validate it against the schema mentioned in the original document.

**Comment by belyrs** [comment]
> Please note. Any changes inside of SDK should be called out, because it needs a release to get reflected for those who want to run this.

Noted. Thank you

**Comment by NiranjanAP** [comment]
> Please note. Any changes inside of SDK should be called out, because it needs a release to get reflected for those who want to run this.

@amarts  you mean call out internally?

**Comment by amarts** [comment]
> @amarts you mean call out internally?

Call out in description of the PR itself, so when we review, and later when we check `git log` we have context of when all a release was required etc.

**Comment by NiranjanAP** [comment]
@adi-a11y  pls review the comments and close this PR

**Comment by belyrs** [comment]
> @adi-a11y pls review the comments and close this PR

Sure, working on it

**Pull request closed without merging**